charts/trino/templates/configmap-worker.yaml

```diff
@@ -57,10 +57,19 @@ data:
     {{- range $configValue := .Values.additionalConfigProperties }}
     {{ $configValue }}
     {{- end }}
+    {{- if .Values.worker.gracefulShutdown.enabled }}
+    shutdown.grace-period={{ .Values.worker.gracefulShutdown.gracePeriod }}
+    {{- end }}
     {{- if .Values.server.workerExtraConfig }}
     {{- .Values.server.workerExtraConfig | nindent 4 }}
     {{- end }}
 
+{{- if .Values.worker.gracefulShutdown.enabled }}
+  access-control.properties: |
```

Comment:
I'm not entirely sure graceful shutdown is the only reason to configure access control on workers. #213 has a more generic approach.

Reply:
See my other comment about failing if both a custom preStop hook is configured, and graceful shutdown is enabled. Maybe the same can be applied here - allow configuring custom access control only if graceful shutdown is not enabled.

Reply:
I have checked the documentation and graceful shutdown is the only place where authorization is mentioned on worker nodes. And basically you can either [set system access control to `allow-all`](https://trino.io/docs/current/admin/graceful-shutdown.html), or to setup [file access control with write access to the `system_information` schema](https://trino.io/docs/current/security/file-system-access-control.html#system-information-rules). I went with the second because I thought a least permissions approach was better. 

In any case, I think I can use the multiple authorization configurations feature and allow other authorization mechanisms alongside this.